README.adoc

```diff
@@ -31,24 +31,39 @@ us on Spectrum,link=https://spectrum.chat/opfab]
 == Introduction
 
 //tag::short_description[]
-OperatorFabric is a modular, extensible, industrial-strength and field-tested
-platform for use in electricity, water, and other utility operations.
+OperatorFabric is a modular, extensible, industrial-strength platform for use in electricity, water, and other utility operations.
 
-* System visualization and console integration
-* Precise alerting
-* Workflow scheduling
-* Historian
-* Scripting (ex: Python, JavaScript)
+It aims to facilitate operational activities for utilities in two ways :
+
+* Centralize realtime business events in a single place to avoid having multiple screens/software, operatorFabric provide 
```

Comment:
```suggestion
* Centralize real time business events in a single place to avoid having multiple screens/software. To do so,  OperatorFabric provides 
```